Library/Homebrew/tap_migrations.rb

```diff
@@ -143,6 +147,7 @@
   "sylpheed" => "homebrew/x11",
   "syslog-ng" => "homebrew/boneyard",
   "tabbed" => "homebrew/x11",
+  "tcpurify" => "homebrew/boneyard",
```

Comment:
I couldn't find any alternative sources for this; the homepage seems AWOL as well.